src/buildblock/error.cxx

```diff
@@ -36,13 +36,6 @@
 
 #include "stir/TextWriter.h"
 
-/* Warning: vsnprintf is only ISO C99. So your compiler might not have it.
-   Visual Studio can be accomodated with the following work-around
-*/
-#ifdef BOOST_MSVC
```

Comment:
you want like it but can we just use
```
#if defined(BOOST_MSVC) && (BOOST_MSVC<1400)
```
I have no clue if VS 2003 would still work, but I don't want to introduce a largely unnecessary versoin-dependency until we jump to C++-11.